libcxx/test/libcxx/selftest/modules/std-and-std.compat-module.sh.cpp

```diff
@@ -12,6 +12,10 @@
 
 // XFAIL: has-no-cxx-module-support
 
+// picolibc does not provide the required timespec_get function, and the
+// "using-if-exists" mechanism apparently did not work here.
```

Comment:
We don't use `using-if-exists` in `ctime.inc`, that might be the problem. Otherwise when `timespec_get` is the only missing part we could `ifdef` that in `ctime.inc` if `using-if-exists` fails.

Reply:
Added a follow up patch https://github.com/llvm/llvm-project/pull/78580